Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -401,6 +404,20 @@ AppDelegateViewModelOutputs {
       .filter { $0 == .tab(.login) }
       .ignoreValues()
 
+    self.goToCreatorMessageThread = deepLink
+      .map { navigation -> (Param, Int)? in
+        guard case let .creatorMessages(projectId, messageThreadId) = navigation else { return nil }
+        return .some((projectId, messageThreadId: messageThreadId))
+      }
+      .skipNil()
+      .switchMap { projectId, messageThreadId in
+        AppEnvironment.current.apiService.fetchMessageThread(messageThreadId: messageThreadId)
+          .demoteErrors()
+          .map { (env: MessageThreadEnvelope) in
```

Comment:
Oh, from the diff it looks like it didn't have the type before?

<img width="369" alt="screen shot 2017-08-03 at 3 46 47 pm" src="https://user-images.githubusercontent.com/3735375/28940540-0ba3dd72-7863-11e7-9e88-34781cf4cae3.png">